staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -101,7 +101,11 @@ func (o *Options) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
 }
 
 // Validate validates the provided options
-func (o *Options) Validate() error {
+func (o *Options) Validate(cmd *cobra.Command, args []string) error {
```

Comment:
```suggestion
func (o *Options) Validate(args []string) error {
```

We're trying to remove the dependency on cobra directly for these.